Library/Formula/phantomjs.rb

```diff
@@ -17,9 +17,13 @@ class Phantomjs < Formula
   depends_on "openssl"
 
   def install
-    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
-              "QT_CFG+=' -openssl -openssl-linked -I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}'"
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs
+    if build.stable?
+      system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
+             "--qt-config", "-I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}"
+    else
+      inreplace "build.py", "/usr/local", HOMEBREW_PREFIX
```

Comment:
@mikemcquaid yes, thanks. I'll update it. Good point.

>As I mentioned on the PhantomJS CI issue: I think being able to use a system Qt (5) for as much as possible would >make life much easier for both PhantomJS and downstream packagers. I wonder if it would be possible to build just >QtWebKit there.

Absolutely. I'll investigate this opportunity.